src/count-cats.js

```diff
@@ -1,6 +1,21 @@
 const CustomError = require("../extensions/custom-error");
 
-module.exports = function countCats(/* matrix */) {
-  throw new CustomError('Not implemented');
-  // remove line with error and write your code here
+module.exports = function countCats(cats) {
+  let total = 0;
+  if (!Array.isArray(cats) || cats.length == 0) {
```

Comment:
NOTE 1 - what's the best way to test the input argument? E.g. how to verity in the input is a non-empty array?